tests/foreman/ui/test_ansible.py

```diff
@@ -229,6 +236,27 @@ def _finalize():
                 (v['Name'], v['Ansible role'], v['Type'], v['Value']) for v in variable_table
             ]
 
+            new_key = gen_string('alpha')
+            session.ansiblevariables.create_with_overrides(
+                {
+                    'key': new_key,
+                    'ansible_role': SELECTED_ROLE[0],
+                    'override': 'true',
+                    'parameter_type': parameter_type,
+                    'default_value': default_value,
+                }
+            )
+            new_value = '["test update"]'
+            # Returns the updated value stored in a variable.
+            update_value = session.host_new.read_update_value(
+                rhel_contenthost.hostname, new_key, new_value
+            )
+            assert new_value in update_value
+            # Revert the updated value to its default state.
+            delete_value = session.host_new.read_value_after_del(rhel_contenthost.hostname, new_key)
```

Comment:
Can you please rename this variable? The name is confusing.
may be `reset_variable_value` ?

Reply:
updated the variable name.